nixos/modules/programs/z.nix

```diff
@@ -0,0 +1,109 @@
+with builtins;
+{ config, lib, pkgs, ... }:
+  let
```

Comment:
```suggestion

{ config, lib, pkgs, ... }:

with builtins;
  let
```

using lib is always preferred over builtin.

Reply:
could you elaborate on this? I don't understand the purpose of this change.

Reply:
The coding style in nixpkgs always puts inputs first before and with.